### PR TITLE
About Page and Story Component Large screen

### LINKED ISSUE #25: OurStory-About

### PR DESCRIPTION
**Pull request by chiturca**
# Description

"pages" folder has been created for new pages and "About" folder implemented to it. Inside, AboutPage component has been added as a container for holding the related components. Story component has been styled with Tailwind css for large screens. ESLint has been added.

## Related Issue

- Fixes #25

## Screenshots
Story

## Type of change

- Enhancement

# How Has This Been Tested?

ESLint

- [x] Tested locally
- [x] Added tests

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] I've run linter, prettier, and removed all `console.log`
- [x] I've reverted all the files back that are not related to the issue (App.js, index.js, ...)

### CODE REVIEW

File: src/pages/About/AboutPage.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+// This component will be used as a container for About Page Components
+
+function AboutPage() {
+  return (
+    <div name="aboutpage">
+      <Story />
+      {/* this part is a placeholder for Team component  */}
+      {/* this part is a placeholder for Testimonials */}
+      {/* this part is a placeholder for Partners */}
+      {/* this part is a placeholder for Call to Action */}
```
Comment (Omer-Amr) [inline src/pages/About/AboutPage.jsx:13]: thanks for adding comment it's really helpful

File: src/pages/About/Story.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            distribution of letters, as opposed to using `Content here, content
+            here`, making it look like readable English.
+          </p>
+        </div>
+      </div>
+    </section>
+  );
+}
+
+export default Story;
```
Comment (Omer-Amr) [inline src/pages/About/Story.jsx:31]: looks very good Miray!

**Review by Omer-Amr — APPROVED** [review]
Appreciate your work 👏🏼💯!

**Comment by batoulst** [comment]
Good job Miray👍

**Review by batoulst — APPROVED** [review]

**Pull request merged**